src/pt/bakai/src/eu/kanade/tachiyomi/extension/pt/bakai/Bakai.kt

```diff
@@ -164,14 +164,23 @@ class Bakai : ParsedHttpSource() {
     override fun pageListParse(document: Document): List<Page> {
         return document.select("div.ipsGrid div.ipsType_center > img")
             .mapIndexed { index, item ->
-                Page(index, "", item.absUrl("data-src"))
+                Page(index, "", item.imgAttr())
             }
     }
 
     override fun imageUrlParse(document: Document): String {
         throw UnsupportedOperationException()
     }
 
+    private fun Element.imgAttr(): String {
+        return when {
+            hasAttr("data-lazy-src") -> attr("abs:data-lazy-src")
+            hasAttr("data-src") -> attr("abs:data-src")
+            hasAttr("data-cfsrc") -> attr("abs:data-cfsrc")
+            else -> attr("abs:src")
+        }.toHttpUrl().toString()
```

Comment:
why?

Reply:
Some images don't have a protocol (http), so they can't be loaded.

In the first commit, I'm checking if there is a protocol prefix and adding it if there isn't, but in the last commit, I noticed that calling toHttpUrl has the same result.